c2corg_ui/static/partials/cards/routes.html

```diff
@@ -23,7 +23,17 @@
             <span class="glyphicon glyphicon-resize-vertical" ng-if="::doc['height_diff_difficulties']"></span>
             <span class="value" ng-if="::doc['height_diff_difficulties']">{{::doc['height_diff_difficulties']}}&nbsp;m</span>
           </span>
-          <div  class="orientations"
+          <span class="slackline-length" ng-if="::doc['route_length'] && cardCtrl.hasActivity(['slacklining'])"
```

Comment:
@stef74 Wouldn't this ``cardCtrl.hasActivity()`` method be useful in your work about card contents?